### PR TITLE
RZ-9295 same approach for the link in the last message and final button

### PR DESCRIPTION
**Pull request by insssomniac**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "1ff-chat-ui",
-  "version": "1.0.46",
+  "version": "1.0.47",
```
Comment (IlievaDayana) [inline package.json:3]: you dont need to update the version, there is a deployment script that does this for you;

File: src/lib/chat-ui.js [hunk 2]
```diff
@@ -501,6 +504,16 @@ const ChatUi = {
     });
   },
 
+  attachLinkEventListener() {
+    const links = document.querySelectorAll('.js-assistant-message a');
+    const link = links[links.length - 1];
```
Comment (IlievaDayana) [inline src/lib/chat-ui.js:509]: why do you select the last anchor tag, if there are a couple anchor tags?

Comment (insssomniac) [reply src/lib/chat-ui.js:509]: Bcs we should attach the handler to it, which sets the flag "chat seen" and close the chat. But should we define the chat as seen, if the link the user clicked is somewhere on the middle or at the beginnig of the chat, and we have other links below?

Comment (insssomniac) [reply src/lib/chat-ui.js:509]: Yes, I've discussed this with PMs. If we have more links comewhere in the chat (in last message or above) - we can allow to visitors just to click to them as it works for regular links, but the last link which is the same as in the button, should be tracked and the chat should be closed.

**Review by IlievaDayana — APPROVED** [review]

**Pull request merged**